Library/Formula/sslscan.rb

```diff
@@ -6,18 +6,12 @@ class Sslscan < Formula
   sha256 "0631713b16cea51df49b9666aa17e742e8177d79e85bdb13f66105657c98f169"
   head "https://github.com/rbsec/sslscan.git"
 
-  bottle do
-    cellar :any
-    sha256 "fc1966c79c103f55cd09b1e9e73b9fa813e115364e2d88d9c5d5a2fc5353fcb2" => :el_capitan
-    sha256 "a550ac845f1b4da274aea5637bc09c5492536bea05f446a0e5aa3dd8be45c35c" => :yosemite
-    sha256 "eda25606cd361b28afbcd64397933d6d22023b821a716aad99f512cc7ef0e838" => :mavericks
-  end
```

Comment:
Please leave this block as is; our bot will update it. To build the formula from source use `brew install --build-from-source` or `brew install -s`.